src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -46,6 +46,11 @@ export const unsavedChangesBadgeStrings = {
     i18n.translate('dashboard.unsavedChangesBadge', {
       defaultMessage: 'Unsaved changes',
     }),
+  getUnsavedChangedBadgeToolTipContent: () =>
+    i18n.translate('dashboard.unsavedChangesBadgeToolTipContent', {
+      defaultMessage:
+        'This dashboard has changed since the last time it was saved. To clear this message save the dashboard.',
+    }),
```

Comment:
```suggestion
    i18n.translate('dashboard.unsavedChangesBadgeToolTipContent', {
      defaultMessage:
        'You have unsaved changes in this dashboard. To remove this label, save the dashboard.',
    }),
```

Source: https://github.com/elastic/kibana/pull/134696#issuecomment-1162280153